Added content and updated README

braden-wilson opened the pull request:
**What does this PR do / why we need it**
All of the new content from the [draft](https://docs.google.com/document/d/1-lSyhEzLaElt2_cGH93vSnOr5nP04f4co8AkywbCsSE/edit#) was added to the website, and the README was filled in with some basic information.

maysunfaisal commented on README.md:
```diff
@@ -1,2 +1,21 @@
+
+## Github Pages
+
+This repo is hosted on github pages at https://braden-wilson.github.io/landing-page.
```
```suggestion
This repo is hosted on github pages at https://devfile.github.io/landing-page.
```

elsony commented on components/index/ValueProps.tsx:
```diff
@@ -0,0 +1,49 @@
+        </GridItem>
+        {index % 2 === 0 ? (
+          <GridItem span={5} style={{ padding: '2rem' }}>
+            picture
```
It may take a while before we can get some graphics created. For the time being, we can just put each value prop as two columns side by side, i.e. remove the picture holder. You can save this placeholder version somewhere so that we can use this layout after we have the pictures available.

elsony commented on pages/index.tsx:
```diff
@@ -24,13 +25,23 @@ const HomePage: React.FC<InferGetStaticPropsType<GetStaticProps>> = () => (
+          padding: '3rem',
+        }}
+      >
+        {ValueProps}
```
May want to reduce the padding spaces between why devfiles and value prop. It is a little too big for the time being. Reduce by 20% will be fine.

elsony commented on components/index/KeyFeatures.tsx:
```diff
@@ -9,84 +7,34 @@ import {
+import keyFeaturesList from '@components/index/keyFeaturesList';
 import cogIcon from '../../public/images/cogs.png';
 
 const KeyFeatures = (
```
With the current 8 key features, it is a bit too much horizontal scrolling before seeing the last item, is it possible to move to another row for items that don't fit the current page size?

elsony commented on components/page/Footer.tsx:
```diff
@@ -35,23 +35,32 @@ const Footer = () => (
+              href="https://www.redhat.com/en"
             >
-              IBM
+              Red Hat
```
Instead of text for the contributors, can we use company logos instead? I am fine with leaving it as a text for the first version and change them to logos in a different PR.

braden-wilson replied:
This can be added in a future PR.

elsony commented on components/starting/StartingNavList.tsx:
```diff
@@ -1,28 +1,42 @@
+      },
+    ],
+    [
+      'Enterprise Architect and Runtime Provider',
```
This header is indented when I test it out for some reason.

braden-wilson replied:
I opened an issue about this in devfile/api#588 and this should be fixed in a future update, but for now there doesn't seem to be any way around it unfortunately.

elsony commented on components/starting/StartingNavList.tsx:
```diff
@@ -1,28 +1,42 @@
     ],
     [
-      'Develop with devfiles',
+      'Technology and Tools Builders',
```
This header is indented when I test it out for some reason.

elsony reviewed (changes requested):
Another thing that I notice is that the top left corner icon should probably be changed to variants of https://github.com/devfile/api/blob/main/docs/icons/2021_Devfile_logo_Config_A.svg (the one with devfile.io text).

elsony commented on components/page/Layout.tsx:
```diff
@@ -11,11 +9,11 @@ import {
 
 import devfileLogo from '../../public/images/devfileLogo.svg';
-import githubLogo from '../../public/images/githubLogo.png';
+import githubLogo from '../../public/images/githubLogo.svg';
```
This file is missing

braden-wilson replied:
These should both be fixed now

elsony commented on components/index/KeyFeatures.tsx:
```diff
@@ -1,92 +1,41 @@
-import enterpriseIcon from '../../public/images/enterprise.png';
-import cogIcon from '../../public/images/cogs.png';
+import type { TitleDescriptionList } from 'custom-types';
+import cogIcon from '../../public/images/cogs.svg';
```
file missing

elsony reviewed (changes requested):
Please make sure things are compiling correctly before requesting a PR review. Are we planning to add PR test to avoid this type of problem? 

Some image files seem to be missing and I also get a compilation error when doing the build:
info  - Using webpack 5. Reason: Enabled by default https://nextjs.org/docs/messages/webpack5
Failed to compile.

./pages/index.tsx:8:61
Type error: Cannot find module '@util/server' or its corresponding type declarations.

   6 | import ValueProps from '@components/index/ValueProps';
   7 | import KeyFeatures from '@components/index/KeyFeatures';
>  8 | import { getTitleDescriptionList, getDescriptionList } from '@util/server';
     |                                                             ^
   9 | 
  10 | import type { TitleDescriptionList } from 'custom-types';
  11 | 
error Command failed with exit code 1.

elsony reviewed (changes requested):
There are 4 comments that have not been addressed:
1. https://github.com/devfile/landing-page/pull/4#pullrequestreview-723839131
2. https://github.com/devfile/landing-page/pull/4#discussion_r683825011 (see image below)
3. https://github.com/devfile/landing-page/pull/4#discussion_r683826664 (see image below)
4. I also notice the github logo does not show (https://github.com/devfile/landing-page/pull/4#discussion_r686153351) (see image below)

![image](https://user-images.githubusercontent.com/7669867/129233442-38cde242-8ae6-4046-be48-03d05ca4635b.png)

maysunfaisal reviewed (commented):
![Screen Shot 2021-08-17 at 12 55 25 PM](https://user-images.githubusercontent.com/31771087/129768531-5c8494ee-2a09-4069-942a-3e7b27a85d71.png)

I would say the features tile card would need to be arranged symmetrically. Probably 2 x 4 format in the above screenshot 🤔

braden-wilson commented:
> There are 4 comments that have not been addressed:
> 
> 1. [Added content and updated README #4 (review)](https://github.com/devfile/landing-page/pull/4#pullrequestreview-723839131)
> 2. [Added content and updated README #4 (comment)](https://github.com/devfile/landing-page/pull/4#discussion_r683825011) (see image below)
> 3. [Added content and updated README #4 (comment)](https://github.com/devfile/landing-page/pull/4#discussion_r683826664) (see image below)
> 4. I also notice the github logo does not show ([Added content and updated README #4 (comment)](https://github.com/devfile/landing-page/pull/4#discussion_r686153351)) (see image below)
> 
> ![image](https://user-images.githubusercontent.com/7669867/129233442-38cde242-8ae6-4046-be48-03d05ca4635b.png)

These have all been addressed.

maysunfaisal reviewed (dismissed):
lgtm, waiting on Elson, Paul

elsony reviewed (dismissed):
Changes look good. Note that the following comments have not been addressed yet:
https://github.com/devfile/landing-page/pull/4#discussion_r683825011
https://github.com/devfile/landing-page/pull/4#discussion_r683826664

Based on previous discussions, those two will be addressed in a different PR so I am marking this one as approved.

schultzp2020 commented on components/starting/SideNavigation.tsx:
```diff
@@ -1,50 +1,61 @@
+        {StartingNavList(navProps).map(
+          ([sectionTitle, pages]): JSX.Element => (
+            <NavExpandable
+              title={sectionTitle as string}
```
Change it back to the `<span>` and add `// @ts-expect-error Does not support span` before the span.

braden-wilson replied:
I added this, it works

elsony reviewed (approved):
The indent problem has been fixed. /lgtm
